www/js/controller.js

```diff
@@ -1050,6 +1053,15 @@ define([
 						this.addNode(this._model.active.getNode(newNode.id));
 						this.setNodeDescription(newNode.id,newNode.variable);
 					}, this);
+					//dynamicList contains those nodes for which prior node UI changes have to be made
+					//Accordingly, make the node dynamic by changing the variable type and setting the accumulator
+					//Also updateNodeView makes sure changes are reflected instantly on the UI
+					var dynamicList = parseObject.dynamicList;
+					array.forEach(dynamicList, lang.hitch(this,function(prior){
+						this._model.authored.setVariableType(prior.id,"dynamic");
+						console.log("prior id", prior.id);
```

Comment:
I think triggering handleVariableType ( triggering event corresponding to changing radio buttons for var type) is not relevant here because dynamicList is collection of multiple prior nodes and all the list nodes have to be updated, let me know what you think